CONTRIBUTING.md

```diff
@@ -194,53 +194,22 @@ to do so. Travis also runs shellcheck.
 
 ## Releasing
 
-To cut a release, push a new tag (versioning discussed below).
-
-### Tagging a release
-
-1. Run `make changes` to review the changes since the last
-   release. Based on the changes, decide what kind of release you are
-   doing (bugfix, feature or breaking).
```

Comment:
This PR should have resolve the issue you had with `make changes`. Can you try it again?

https://github.com/digitalocean/doctl/pull/1576

Reply:
Works for me :+1: Thanks.
Updated the doc in [f595a6a](https://github.com/digitalocean/doctl/pull/1577/commits/f595a6a427fedf50705c0fc6c7d625e336a94d7a)